content/200-concepts/100-components/250-preview-features/050-client-preview-features.mdx

```diff
@@ -12,6 +12,7 @@ The following [Preview](/about/prisma/releases#preview) feature flags are availa
 - [`referentialIntegrity`](/concepts/components/prisma-schema/relations/referential-integrity) since release [3.1.1](https://github.com/prisma/prisma/releases/tag/3.1.1) - [Submit feedback](https://github.com/prisma/prisma/issues/9380)
 - [`fullTextIndex`](/concepts/components/prisma-schema/indexes#full-text-indexes-mysql-and-mongodb) since release [3.6.0](https://github.com/prisma/prisma/releases/tag/3.6.0) - [Submit feedback](https://github.com/prisma/prisma/issues/10539)
 - [`metrics`](/concepts/components/prisma-client/metrics) since release [3.15.0](https://github.com/prisma/prisma/releases/tag/3.15.0) - [Submit feedback](https://github.com/prisma/prisma/issues/13579)
+- [`filteredRelationCount`](/concepts/components/prisma-client/aggregation-grouping-summarizing#filter-the-related-records) since release [4.1.0](https://github.com/prisma/prisma/releases/tag/4.1.0) **need to add feedback link**]
```

Comment:
Are you still waiting for this issue link?

Reply:
Yes I am - it's on my to-do list for release.

Reply:
I'll leave this unresolved as an extra reminder.

Reply:
Those issues are usually created close to when the preview feature is released, but now should be fine - someone just needs to create it by a similar template like the existing ones and apply the appropriate labels as well.

Reply:
I've added the feedback link now, so I'm closing this conversation